includes/class-database-cleanup.php

```
@@ -19,634 +19,636 @@
+		 * @since 1.4.0
+		 * @param string $method The static method name to invoke.
+		 * @param mixed  ...$args Arguments forwarded to the method.
+		 * @return mixed The invoked method's return value, or a `WP_Error` if the method returned `false`.
+		 */
+		public static function invoke_cleanup_method( $method, ...$args ) {
+			$res = self::$method( ...$args );
+			if ( false === $res ) {
+				return new WP_Error( 'db_cleanup_failed', sprintf( '%s failed', $method ) );
+			}
+			return $res;
 		}
```

PHPMD (2.15.0)

[error] The method invoke_cleanup_method is not named in camelCase. (undefined)

(CamelCaseMethodName)